lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -1515,6 +1521,20 @@ extension DomImageDataExtension on DomImageData {
   @JS('data')
   external JSUint8ClampedArray get _data;
   Uint8ClampedList get data => _data.toDart;
+  external int get width;
+  external int get height;
+  external PredefinedColorSpace get colorSpace;
+}
+
+@JS('Uint8ClampedArray')
+@staticInterop
+class JSUint8ClampedArrayData {
```

Comment:
<img width="787" alt="Screenshot 2024-08-12 at 9 15 18 AM" src="https://github.com/user-attachments/assets/e5fb7ddb-8dfe-4a37-bc28-a527988c1e6f">

@mkustermann  I get this error when trying to use that constructor. Is this available in dart 3.2? It looks like the dart sdk is pinned a 
few versions back, can't use current js_interop extension classes either. Not sure if it's ok to bump it to get newer stuff, but it would really be nice to bump it.

```yaml
name: ui
publish_to: none

# Keep the SDK version range in sync with pubspecs under web_sdk
environment:
  sdk: '>=3.2.0-0 <4.0.0'
```